README.md

```diff
@@ -49,6 +49,23 @@ How To Get
 Check the [Github Project](https://github.com/nealey/hdjd) for the canonical source repo.
 
 
+How to Build
+------------
+
+Install Dependencies:
+
+* Debian / Ubuntu
+
+ apt-get install libusb-1.0.0-dev libasound2-dev
+
+* Rocky / CentOS:
+
+ yum install libusb-devel alsa-lib-devel
+
+Build hddj:
```

Comment:
Please make this an L3 heading as well, so it's clear it's separate from the OS instructions